src/adapters/http/client.ts

```diff
@@ -0,0 +1,29 @@
+import Adapter from "../../lib/adapter.js";
+
+const axios = require('axios');
+
+class HttpClientAdapter extends Adapter {
+
```

Comment:
You need to cache the client connection, because there could be more than one client servers in spec